onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -732,9 +745,9 @@ def _where_input_wrangler(
     ),
     TorchLibOpInfo("gather", core_ops.aten_gather),
     TorchLibOpInfo("ge", core_ops.aten_ge),
-    # TorchLibOpInfo("greater_equal", core_ops.aten_greater_equal),  # no test case in OPS_DB
```

Comment:
Do we need to add bool test case?

Reply:
They are added as ge_bool etc. Am I missing something?

Reply:
I asked because I don't see extra_opinfo.py is modified.

Reply:
I see. I just duplicated the test. They include test cases with bool inputs.